CONTRIBUTING.md

```diff
@@ -1,21 +1,129 @@
-# Contributing
+# How to contribute
 
-## Evolving the Format of the Forecast Data
+Soaringmeteo team is glad you are reading this, because we always need volunteers
+to keep this awesome service up to date.
```

Comment:
```suggestion
to keep this service up to date.
```

I think we should adopt a more neutral tone.

Reply:
done